Missionframework/KPGUI/KPGUI_defines.hpp

```diff
@@ -142,6 +142,16 @@
 #define KP_WIDTH_VAL_LP                 0.2
 #define KP_HEIGHT_VAL_LP                (1 - 2 * KP_Y_VAL_LP - KP_HEIGTH_TITLE - KP_HEIGTH_BUTTON - 2 * KP_SPACING_Y)
 
+/*
+    --- Right panel dialog components ---
+*/
```

Comment:
```suggestion
    (X from 0.7975 - 0.9975, Y from 0.0025 - 0.9975)
*/
```
For the left panel dialog:
```
    (X from 0.0025 - 0.2025, Y from 0.0025 - 0.9975)